ZRC_evm/exec.h

```diff
@@ -10,12 +10,16 @@
 
 
 #ifndef UnarOperationString
-	#define UnarOperationString "\nUnary operation:\n\n"
+	#define UnarOperationString "\nUnary operations:\n\n"
 #endif
 
 #ifndef BinaryOperationString
-	#define BinaryOperationString "\nBinary operation:\n\n"
+	#define BinaryOperationString "\nBinary operations:\n\n"
 #endif
+#ifndef TriganometricOperationString
+	#define TriganometricOperationString "\nTriganometric operations:\n\n"
```

Comment:
Why?

Reply:
Trigan operations are separated part of math :O